assets/js/modules/analytics-4/datastore/report.js

```diff
@@ -119,8 +119,8 @@ const fetchGetReportStore = createFetchStore( {
 		} = normalizeReportOptions( options );
 
 		invariant(
-			metrics.length,
-			'Requests must specify at least one metric for an Analytics 4 report.'
+			metrics?.length || dimensions?.length,
```

Comment:
I've loosened this to accommodate the report we make to check gathering data state, as we don't need to specify a metric for it.

Reply:
Have reverted this change as discussed.